requirements.txt

```diff
@@ -23,3 +23,6 @@ piq
 git+https://github.com/ChaoningZhang/MobileSAM.git
 Ninja
 lpips
+pytest
```

Comment:
We could have a requirements.txt for dev, like suggested here: https://stackoverflow.com/questions/42179763/can-a-package-be-required-only-for-tests-not-for-installation
@beniz what do you think?